metadata.yaml

```diff
@@ -50,3 +50,6 @@ provides:
     interface: lxd-https
   metrics:
     interface: lxd-metrics
+requires:
```

Comment:
I really don't like the wording of `requires`, all the Charm docs suggest that those are necessary for the proper operation of the charm, which is definitely not the case here.

Maybe file a Juju bug to have them either rename this to `consumes` or something, or add another such keyword to properly distinguish relations which are actually required to those that merely enhance the charm when connected?

Reply:
https://juju.is/docs/sdk/relations mentions an `optional` keyword that is meant to reduce the confusion of the `requires` relations. I'm adding it in PR #32.